packages/server/src/utils.ts

```diff
@@ -1,14 +1,29 @@
 import dayjs from '~/lib/dayjs';
 
-export function getActiveUntilDate(oldActiveUntil: Date, anchorDate: Date): Date {
-  const anchorDay = Math.min(dayjs(oldActiveUntil).add(1, 'month').endOf('month').date(), dayjs(anchorDate).date());
-  return dayjs(oldActiveUntil).add(1, 'month').set('date', anchorDay).toDate();
+export function getPeriodFromAnchorDate(someDateInPeriod: Date, _anchorDate: Date): { start: Date; end: Date } {
```

Comment:
@anbraten Why do you have the underscore before anchorDate parameter?

Reply:
had a var called `anchorDate` converted using dayjs in the function before.